db/schema.rb

```diff
@@ -1144,6 +1144,7 @@
     t.tsvector "tsv"
     t.string "video_url"
     t.boolean "mandatory_answer", default: false, null: false
+    t.string "validator"
```

Comment:
Since we changed the date of the `mandatory_answer` migration, it looks like we need to change the order here as well :thinking: so it's consistent with what we get when running `rake db:drop db:create db:migrate`. Here's a suggestion:

```ruby
    t.string "validator"
    t.boolean "mandatory_answer", default: false, null: false
```

Reply:
Done. Nice catch! 🙏🏼